dags/oup/parser.py

```diff
@@ -54,8 +58,20 @@ def _get_journal_doctype(self, article: ET.Element):
             journal_doctype = self.article_type_mapping[journal_doctype_raw]
         except KeyError:
             journal_doctype = "other"
+        if "other" in journal_doctype:
             self.logger.warning(
                 "There are unmapped article types for article",
                 journal_doctype=journal_doctype,
             )
         return journal_doctype
+
+    def _get_arxiv_eprints(self, article: ET.Element):
+        arxivs_raw = get_text_value(
+            article.find("front/article-meta/article-id/[@pub-id-type='arxiv']")
+        )
+        if not arxivs_raw:
+            return
+        arxiv_eprint = arxivs_raw.lower().replace("arxiv:", "")
```

Comment:
are you sure arxiv id is always like that? on inspire we have some different versions https://github.com/inspirehep/inspire-schemas/blob/32d9168cb17d6388d67aad311dff93dac99b15b0/inspire_schemas/utils.py#L102-L105
and recently we've been relaxing our schema to accept another pattern

Reply:
The format of arxiv is not specified in this change. It is just "arxiv:" part removal. If this part is not in arxiv string, the string of arxiv value won't be changed